astropy/wcs/wcs.py

```diff
@@ -45,14 +45,8 @@
 # LOCAL
 from astropy import log
 from astropy.io import fits
-from . import _docutil as __
```

Comment:
Not really a review comment, but I never understood the rationale for importing things this way. Is there a good reason for using `__` as the module name?

Reply:
I have no idea and I found it veeeery confusing